mon-pix/app/styles/components/_recover-account-student-information-form.scss

```
@@ -0,0 +1,47 @@
+.recover-account-student-information-form {
+  position: relative;
+
+  &__fields {
+    max-width: 600px;
```

Suggestion d'après Abstract :
réduire la largeur des inputs. ça implique de retoucher l'emplacement du tooltip.
Si tu veut qu'on en discute, dis moi :) 
```suggestion
    max-width: 500px;
```

---

Idem, ici je vais faire la review avec Manu cet aprèm je pense qu'on peut lui poser directement la question :)